### PR TITLE
Add FederatedCode integration

### PR DESCRIPTION
**Pull request by keshav-space**
- Related https://github.com/aboutcode-org/federatedcode/issues/14 and https://github.com/aboutcode-org/federatedcode/issues/47

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -63,9 +63,12 @@ install_requires =
     matchcode-toolkit == 7.0.0
     purl2vcs == 2.0.0
     univers == 30.12.1
-    scancodeio @ git+https://github.com/aboutcode-org/scancode.io.git@3eeb0c040c6077d773680b59da6fa283ba1500bf
+    scancodeio @ git+https://github.com/aboutcode-org/scancode.io@bumb-hashid
```
Comment (pombredanne) [inline setup.cfg:66]: @keshav-space are you sure you branch also covers the commit above?
@JonoYang ping

Comment (pombredanne) [reply setup.cfg:66]: @keshav-space https://github.com/aboutcode-org/scancode.io/pull/1479 is merged now. Can you update your PR and also take into account that: https://github.com/aboutcode-org/scancode.io/commit/3eeb0c040c6077d773680b59da6fa283ba1500bf and then sync with @JonoYang  ?

**Review by pombredanne — CHANGES_REQUESTED** [review]
Looking great ... Can you confirm that you change is not breaking the head with @JonoYang changes for scancode.io?

File: setup.cfg (hunk 1 above)
Comment (keshav-space) [reply setup.cfg:66]: @pombredanne https://github.com/aboutcode-org/scancode.io/commit/3eeb0c040c6077d773680b59da6fa283ba1500bf belongs to https://github.com/aboutcode-org/scancode.io/pull/1466 which is already merged. I will update the PR to use the scancode.io@main

**Review by JonoYang — CHANGES_REQUESTED** [review]
@keshav-space The code looks fine, just needs some tests for the new API endpoints. 👍

**Comment by keshav-space** [comment]
> The code looks fine, just needs some tests for the new API endpoints. 👍

@JonoYang Done!

**Comment by JonoYang** [comment]
@keshav-space Thanks! merging

**Pull request merged**